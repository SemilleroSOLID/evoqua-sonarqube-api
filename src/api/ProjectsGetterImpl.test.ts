import { ProjectsGetterImpl } from ".";

const { SONAR_URL, SONAR_USERNAME, SONAR_PASSWORD } = process.env;
const api = new ProjectsGetterImpl(SONAR_URL!);

beforeAll(() => api.login(SONAR_USERNAME!, SONAR_PASSWORD!));

it('getProjects', async () => {
  const projects = await api.getProjects();
  expect(projects).toMatchSnapshot();
});
