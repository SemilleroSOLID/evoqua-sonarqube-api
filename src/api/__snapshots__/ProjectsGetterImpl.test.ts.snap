// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getProjects 1`] = `
Array [
  Object {
    "key": "CvLAC-scraper",
    "name": "CvLAC-scraper",
  },
  Object {
    "key": "LogisticApp",
    "name": "LogisticApp",
  },
  Object {
    "key": "SCAPE-app",
    "name": "SCAPE-app",
  },
  Object {
    "key": "TallerPSP01",
    "name": "TallerPSP01",
  },
  Object {
    "key": "jlopezg8:TallerPSP1",
    "name": "TallerPSP1",
  },
]
`;
